quotefault/__init__.py

```diff
@@ -304,53 +299,60 @@ def review_submit(report_id, result):
     Called when Admin decides on a quote being hidden or kept
     """
     metadata = get_metadata()
-    result = int(result)
-    if metadata['is_admin']:
-        # 1 = Keep, 0 = Hide
-        if result:
-            report = Report.query.get(report_id)
+    report = Report.query.get(report_id)
+    if report and metadata['is_admin']:
+        if result == "keep": 
             report.reviewed = True
             db.session.commit()
             flash("Report Completed: Quote Kept")
-        else:
-            report = Report.query.get(report_id)
+        elif result == "hide":
             report.quote.hidden = True
             report.reviewed = True
             db.session.commit()
             flash("Report Completed: Quote Hidden")
+        else:
+            abort(400)
         return redirect('/review')
-    return redirect('/')
+    elif report:
+        abort(403)
+    abort(400)

```

Comment:
This is kinda confusing. Instead of the nested ifs, could you do:
```
if not admin:
	403

if not report:
	404

if hide:
elif keep:
else:
```